src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -1729,6 +1729,31 @@ contract AllocationManagerUnitTests_ModifyAllocations is AllocationManagerUnitTe
         }
     }
 
+    function test_CannotIncludeBeaconChainETHStrategy() public {
```

Comment:
For sanity can we add a test that attempts to slash beaconChainETH in an operatorSet and does not? We need to ensure no storage mutated or events emitted with beaconChainETH on a slash